### PR TITLE
WIP: Add comment support

### LINKED ISSUE #77: Comments support

### PR DESCRIPTION
**Pull request by PrajwalM2212**
fixes #77

### CODE REVIEW

**Comment by uiri** [comment]
Hi Prajwal,

Thank you for your pull request! But this should really be part of a decoder. One of the reasons I adopted the decoder/encoder model was to avoid an explosion of parameters to `dump` / `load` and `dumps` / `loads`.

Also, I don't think your test is quite right. The point of saving comments is not to add them to the dictionary as key-value pairs to the dictionary that `loads` returns but to make sure that they roundtrip through loads and dumps without modification.

File: tests/test_api.py [hunk 1]
```diff
@@ -6,14 +6,21 @@
 from decimal import Decimal
 
 from toml.decoder import InlineTableDict
-
+from toml.decoder import loads
```
Comment (uiri) [inline tests/test_api.py]: nit: this import is unnecessary. `toml` is imported above, so you can use `toml.loads`

Comment (uiri) [inline tests/test_api.py]: nit: this import is unnecessary. `toml` is imported above, so you can use `toml.loads`

File: toml/decoder.py [hunk 2]
```diff
@@ -43,7 +43,6 @@ def _getpath(p):
 except NameError:
     FNFError = IOError
 
-
```
Comment (uiri) [inline toml/decoder.py]: nit: whitespace change

**Comment by PrajwalM2212** [comment]
@uiri Yes, I totally missed the encoder part. 
So would this approach be okay ?
1.  load comments into a dictionary like I have done here by a key like 'comment_0', 'comment_1' etc.
2. dump by converting these 'comment_0', 'comment_1' into `# comment `

**Pull request closed without merging**